Templates/Portfolio/Objects/Lists/Prosjektstatus.xml

```diff
@@ -1,4 +1,4 @@
-<pnp:ListInstance Title="{resource:Lists_ProjectStatus_Title}" Description="" DocumentTemplate="" TemplateType="100" Url="{resource:Lists_ProjectStatus_Url}" EnableVersioning="true" MinorVersionLimit="0" MaxVersionLimit="5000" DraftVersionVisibility="0" ContentTypesEnabled="true" RemoveExistingContentTypes="true" EnableFolderCreation="false" EnableAttachments="true" 
+<pnp:ListInstance Title="{resource:Lists_ProjectStatus_Title}" Description="" DocumentTemplate="" TemplateType="100" Url="{resource:Lists_ProjectStatus_Url}" EnableVersioning="true" MinorVersionLimit="0" MaxVersionLimit="5000" DraftVersionVisibility="0" ContentTypesEnabled="true" RemoveExistingContentTypes="false" EnableFolderCreation="false" EnableAttachments="true" 
```

Comment:
We might run into an issue on new installs where "Item" till be a selectable content type next to "project status"?

Reply:
The current issue is that it removes all custom content types though, since we only have one binding. So if they have created custom content types without binding them in a template, they are removed. That's what the fix is for. Is item default even when only one content type is bound?

Reply:
Yup, I understand.... Yes, Item will be left there unless we set RemoveExistingContentTypes=true. But perhaps we can specify the Item content type and Remove=true?

https://github.com/pnp/PnP-Provisioning-Schema/blob/master/ProvisioningSchema-2019-03.md#contenttypebinding

Reply:
Alright, this should do the trick. Did the same for the Prosjekter list